### DISCUSSION TITLE
Editable Speaker names

### DISCUSSION THREAD

**Discussion by dancasiele** [kaixxx/noScribe#30] [Ideas]
Es wäre toll, wenn die Sprecher umbenennt werden könnten.

**Comment by kaixxx** [comment]
Ja, gute Idee. Ich habe auch schon darüber nachgedacht, wie sich das in den noScribe Editor integrieren ließe. Kommt auf die Liste. 

Hast du schon noScribe ver. 0.4b ausprobiert? Mac oder Windows? Mich würde interessieren, ob es funktioniert.

**Reply by dancasiele** [reply]
Hallo kaixxx

Ja habe ich ausprobiert (Windows). Es funktioniert sehr gut, auch der Editor funktioniert gut. Echt super Tool.
Eine Frage hätte ich noch:
Findet beim benutzen von noScribe irgendeine Datenübertragung nach Aussen statt, oder funktioniert das alles lediglich lokal und sogar offline? Könnte ich damit auch vertrauliche Daten transkribieren?

**Reply by dancasiele** [reply]
Sorry für meine Frage, hab gerade gesehen, dass die Antwort dazu bereits im Beschrieb steht.

**Comment by kaixxx** [comment]
Kein Problem. Es war ein zentraler Anlass für die Entwicklung von noScribe, dass keine Informationen nach aussen fliessen sollten. Clouddienste zur Transkription gibt es ja schon einige. Probier doch gerne man noScribe mit deaktivierter Internetverbindung aus. 
Schön zu hören, dass die neue Version funktioniert.
